src/js/components/sharedComponents/checkbox/CheckboxOnePrimary.jsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * CheckboxOnePrimary.jsx
```

Comment:
Let's change the name from CheckboxOne to something more descriptive.  A year from now no one is going to know what treatment #1 means.  This treatment uses accordions so AccordionCheckbox could be a possible name.

Reply:
Renamed to Accordion Checkbox!